iceoryx_examples/iceensemble/README.md

```diff
@@ -0,0 +1,40 @@
+# iceensemble
+
+## Introduction
+
+A common use case is that often we have multiple sensors sending data of the same type, e.g. LIDAR data, and a subscriber is interested in the data of all those sensors.
+
+This example demonstrates
+
+1. how you can run multiple publisher applications publishing on the same topic (n:m communication)
+2. how to communicate between C and C++
+
+!!! warning
+    If you have built iceoryx with the CMake flag `ONE_TO_MANY_ONLY` this example will not run
```

Comment:
Is this by default on or do we have to explicitly activate this? Could you please mention this also in the markdown file.

Reply:
Added